libsolidity/codegen/ABIFunctions.h

```diff
@@ -231,6 +231,11 @@ class ABIFunctions
 	/// Only works for memory arrays and storage arrays that store one item per slot.
 	std::string nextArrayElementFunction(ArrayType const& _type);
 
+	/// @returns the name of a function using in encoding that stores the length
```

Comment:
`function using in encoding`? 

Is this meant to be used in encoding?

Reply:
ah right, should be "used in encoding" ("for" encoding? "during" encoding?)